recipes-wpe/wpeframework/wpeframework-rdkservices_git.bb

```diff
@@ -1,13 +1,14 @@
 SUMMARY = "WPEFramework rdkservices"
```

Comment:
Can we make a bump a separate commit?

Reply:
will do after do the settings/cleanup of thunder modules and plugins